app/models/card.rb

```diff
@@ -11,7 +11,7 @@ def self.random
   end
 
   def self.due_today
-    where("review_date < ?", DateTime.now)
+    where("review_date <= ?", DateTime.now.to_date)
```

Comment:
@davetoxa meant to remove that - was trying to debug some timezone issues revealed by the tests